src/solver/conv_winoRxS_f2x3.cpp

```diff
@@ -602,22 +607,40 @@ ConvBinWinogradRxSf2x3::GetSolution(const ConvolutionContext& params,
     };
     kernel.comp_options = options.GenerateFor(kbp::GcnAsm{});
 
+    std::string kernel_name    = "miopenSp3AsmConv";
+    std::string kernel_file    = "Conv_Winograd";
+    std::string kernel_postfix = "_v21_1_0_gfx9";
+
+    if(params.IsFp32())
+        kernel_postfix += "_fp32";
+    else
+    {
+        kernel_postfix += "_fp16";
+        const auto name = params.GetStream().GetDeviceName();
+        if(StartsWith(name, "gfx906") || StartsWith(name, "gfx908"))
+            kernel_postfix += "_dot2_edc";
+        else
+            kernel_postfix += "_pk";
```

Comment:
This is pure FP16 kernel. We are not using pure FP16 kernels. Please disable and remove.